testing_and_setup/compass/ocean/global_ocean/cull_mesh.py

```diff
@@ -194,8 +199,12 @@ def removeFile(fileName):
     # cull the mesh based on the land mask
     # Run command is:
     # ./MpasCellCuller.x  base_mesh.nc culled_mesh_preliminary.nc -m land_mask_final.nc
-    args = ['./MpasCellCuller.x', 'base_mesh.nc', 'culled_mesh_preliminary.nc',
-            '-m', 'land_mask_final.nc']
+    if options.preserve_floodplain:
+      args = ['./MpasCellCuller.x', 'base_mesh.nc', 'culled_mesh_preliminary.nc',
+              '-m', 'land_mask_final.nc', '-p', 'base_mesh.nc']
+    else:
+      args = ['./MpasCellCuller.x', 'base_mesh.nc', 'culled_mesh_preliminary.nc',
+              '-m', 'land_mask_final.nc']
     print("running {}".format(' '.join(args)))
```

Comment:
Should we also inject the bathymetry here, similar to what is done for `build_mesh.py`?

Reply:
We take care of that here: 		
https://github.com/MPAS-Dev/MPAS-Model/blob/c0cbf0cbf5e17fa49eefb85418cfc377c894f22e/testing_and_setup/compass/ocean/coastal/USDEQU120cr10rr2/build_mesh/config_culled_mesh.xml#L22-L24

Reply:
Does that work okay?

Reply:
I'm going to verify on the squashed commit-- I'm thinking it was due to some environment-based errors and if it works for you I would be surprised if it doesn't work for me on this last check.

Reply:
@pwolfram, definitely let me know.

Reply:
We are good-- thanks!